docs/src/Link.doc.js

```diff
@@ -138,6 +148,51 @@ card(
   />
 );
 
+card(
+  <Example
+    id="tab"
+    description={`
+    Use accessibilitySelected and role when using it as a Tab.
+  `}
+    name="Accessible Tab Link"
+    defaultCode={`
+function TabExample() {
+  const [activeIndex, setActiveIndex] = React.useState(0);
+  return (
+    <Box display="flex" alignItems="center" role="tablist">
+      {['Boards', 'Pins'].map((text, index) => (
+        <Box
+          color={index === activeIndex ? "darkGray" : undefined}
+          display="inlineBlock"
+          key={text}
+          rounding="pill"
+        >
+          <Link
+            accessibilitySelected={index === activeIndex}
+            hoverStyle="none"
+            href="#"
```

Comment:
@jackhsu978 could we change this to an actual link? It allows for opening a link in a new tab with Right click, open in new tab (a11y)

Reply:
will do

Reply:
updated